src/modal/index.unit.tsx

```diff
@@ -15,6 +15,12 @@ const defaultProps = {
 }
 
 describe('Modal', () => {
+  // TODO: mount method will be fixed with forwardRef new feature in Enzyme v3.5
+  // As said in this post:
+  // https://www.bountysource.com/issues/
+  // 56657445-forwardref-enzyme-internal-error-unknown-node-with-tag-14
+
+/*
```

Comment:
Enzyme 3.5 is out, you can update package.json, and uncomment the test
https://github.com/airbnb/enzyme/blob/master/CHANGELOG.md